src/Parser/HolyPriest/Modules/PriestCore/Serendipity.js

```diff
@@ -0,0 +1,227 @@
+import React from 'react';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import { formatPercentage, formatNumber } from 'common/format';
+
+// dependencies
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+import SPELLS from 'common/SPELLS';
+import Module from 'Parser/Core/Module';
+
+const HOLY_WORD_SPELL_IDS = [
+  SPELLS.HOLY_WORD_SERENITY.id,
+  SPELLS.HOLY_WORD_SANCTIFY.id,
+];
+
+// This module might contain too much information but to rewrite some of the output
+// in this file would entail a lot of redundant copying (aka violating the "DRY"
+// coding mindset). Let me @enragednuke if you think I should move stuff out.
+
+// Currently contains:
+// - "Wasted Serendipity" statistic
+// - "Fully missed holy word casts (via Serendipity)" suggestion
+// - "Holy Priest T20 2P Bonus" item
+class Serendipity extends Module {
+  static dependencies = {
+    combatants: Combatants,
+  }
+
+  // Holy Word reduction spells (aka things that apply Serendipity)
+  serendipityProccers = {
+    [SPELLS.GREATER_HEAL.id]: SPELLS.HOLY_WORD_SERENITY.id,
+    [SPELLS.FLASH_HEAL.id]: SPELLS.HOLY_WORD_SERENITY.id,
+    [SPELLS.PRAYER_OF_HEALING.id]: SPELLS.HOLY_WORD_SANCTIFY.id,
+  }
```

Comment:
Binding Heal isn't that special of a case, and this structure is barely used. Rethink the structure of what you're doing here to allow you to cut down on duplicated code below.

Reply:
This would be solved by my proposal to split this module into two sub-modules. See my first comment on this PR for what I mean by that. I'm starting to think more and more that it would be better but still would like an extra opinion.

Reply:
Yeah, that may be the best solution, if you're feeling like a refactor.